### ISSUE TITLE
Log exception in Webhook.process?

### ISSUE THREAD

**Issue by blueyed** [pinax/pinax-stripe#370]
Do you think it makes sense to log exceptions in https://github.com/pinax/pinax-stripe/blob/34e6a3cec7a7df2d60c5b2696066c8dba8947e75/pinax/stripe/webhooks.py#L95-L96?

It will create an EventProcessingException object already (via https://github.com/pinax/pinax-stripe/blob/34e6a3cec7a7df2d60c5b2696066c8dba8947e75/pinax/stripe/actions/exceptions.py#L7-L23), but I think it would be good to use `logger.exception` here, for more visibility (server logs etc).

Alternatively we could use the post-save signal on EventProcessingException to do it manually.

**Comment by blueyed** [comment]
It makes a lot of sense, since you might be unable to create a `EventProcessingException` in case of a IntegrityError ('duplicate key value violates unique constraint'), where trying to create it could raise `TransactionManagementError("An error occurred in the current transaction. You can't execute queries until the end of the 'atomic' block.",)`.

**Comment by paltman** [comment]
Are you answering your own question and therefore we can close this?

**Comment by blueyed** [comment]
No, I am rather giving a reason for changing this in the comment.

**Comment by paltman** [comment]
So you are saying there are some exceptions that we'd be prevented from storing in the database?

I'd like to store as much in the database as possible, and only when all else fails, use the logger.

**Comment by paltman** [comment]
I think we should still try to log the exception in the database.  If you want to also add a `logger` line for server logs prior to trying to log into the database I'm fine with that
